bokehjs/src/lib/models/formatters/log_tick_formatter.ts

```diff
@@ -42,10 +44,19 @@ export class LogTickFormatter extends TickFormatter {
 
     const base = this.ticker != null ? this.ticker.base : 10
 
+    const translate: { [id: string]: string; } = { 0:"\u2070", 1:"\u00b9", 2:"\u00b2", 3:"\u00b3", 4:"\u2074", 5:"\u2075", 6:"\u2076", 7:"\u2077", 8:"\u2078", 9:"\u2079", "+":"\u207a", "-":"\u207b" }
```

Comment:
@micrenda if we are going to do both tick formatters in this PR, the `translate` value should be moved to a single place and imported as necessary rather than duplicated